epictrack-api/src/api/models/staff_work_role.py

```diff
@@ -47,3 +47,28 @@ def as_dict(self):  # pylint:disable=arguments-differ
     def find_by_work_id(cls, work_id: int):
         """Return by work id."""
         return cls.query.filter_by(work_id=work_id)
+
+    @classmethod
+    def find_by_role_and_work(cls, work_id: int, role_id):

```

Comment:
would it be good to take the first matched result where this function is being used. This can act as a function which returns the StaffWorkRole for a given work and role id.

Reply:
Okay, I will remove first() and do all() instead